web_src/js/index.js

```diff
@@ -1232,6 +1232,46 @@ function initWikiForm() {
         'clean-block', 'preview', 'fullscreen', 'side-by-side']
     });
     $(simplemde.codemirror.getInputField()).addClass('js-quick-submit');
+
+    setTimeout(() => {
```

Comment:
How about to create a wiki.js and move PR related functions to that file?

Reply:
should this still be compiled into the same lines?
(as this need to be loaded after simpleMDE is ready)

Reply:
line 1272 `sideBySideChanges = 10;` changes variable from line 1137 `let sideBySideChanges = 0;

Reply:
when all simpeMDE related stuff is in there (line 1146 has FIXME and will be fixed in #7390 or #8891), i will add all wiki related simpleMDE changes to simpleMDE element near line 1304.
May i place the simpleMDE stuff into an extra file

Reply:
We could certainly start lazy-loading simpleMDE if it's only used in a few places.